torch/jit/__init__.py

```diff
@@ -1978,9 +1935,15 @@ def register_all(mod):
 
     for builtin, aten_op in builtin_ops:
         _builtin_table[id(builtin)] = aten_op
-    if not PY2:
-        _builtin_table[id(math.gcd)] = "aten::gcd"
-        _builtin_table[id(math.isfinite)] = "aten::isfinite"
+    math_ops = [x for x in dir(math) if callable(getattr(math, x))]
+    unimplemented_math_ops = ["fsum", "isclose", "trunc", "hypot", "log2"]
+    special_math_ops = ["remainder"]    # We bound this to aten::mathremainder.
+    for op in math_ops:
+        if op == "__loader__":  # Weird windows behavior...
```

Comment:
reason not to put op in unimplemented_math_ops ?

Reply:
Same reasoning as above I think.